pkgs/os-specific/linux/keymash/default.nix

```diff
@@ -0,0 +1,37 @@
+{ stdenv, lib, fetchFromGitea, kernel, kmod }:
+let
+  version = "0.1.1";
+in
+stdenv.mkDerivation {
+  pname = "keymash";
+  version = "${version}-${kernel.version}";
+
+  src = fetchFromGitea {
+    domain = "git.bsd.gay";
+    owner = "fef";
+    repo = "keymash";
+    rev = "v${version}";
+    hash = "sha256-sET3NKH8sS9tIvf68nLiNBVQ1so9bISim0Nxr4kd/0I=";
+  };
+
+  hardeningDisable = [ "pic" "format" ];
+  nativeBuildInputs = [ kmod ] ++ kernel.moduleBuildDependencies;
+
+  makeFlags = kernel.makeFlags ++ [
+    "KERNELRELEASE=${kernel.modDirVersion}"
+    "KERNEL_DIR=${kernel.dev}/lib/modules/${kernel.modDirVersion}/build"
+    "INSTALL_MOD_PATH=$(out)"
+  ];
+
+  meta = with lib; {
+    longDescription = ''
+      The gay keymash device driver!
+      /dev/keymash provides an endless supply of lowercase characters from the QWERTY home row in a cryptographically secure random order.
```

Comment:
This is not true as stated in the project's README.

Furthermore:
> The license already disclaims any warranty, but i would like to make it crystal clear that i literally have no idea how to use a computer. Assume this module will completely brick your system.

Reply:
> /dev/keymash ... It provides an endless supply of lowercase characters from the QWERTY home row in a cryptographically secure random order¹

is stated in the README

Reply:
> /dev/keymash ... It provides an endless supply of lowercase characters from the QWERTY home row in a cryptographically secure random order¹.

is stated in the README